src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -265,9 +272,16 @@ public void HideAllPopUp(object sender)
             // First make sure workspace level popups are hidden
             if (InCanvasSearchBar.IsOpen || ContextMenuPopup.IsOpen)
             {
-                ShowHideContextMenu(ShowHideFlags.Hide);
+                ShowHideContextMenu(ShowHideFlags.Hide);               
```

Comment:
please remove extra trailing spaces

Reply:
removed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13717/commits/407c470ede6a2937d2cad92d3989cf1bb8339484